### PR TITLE
adds homepage links and help text

### LINKED ISSUE #12: Visual design/branding for Bento home
tbd

### PR DESCRIPTION
**Pull request by mejackreed**
Fixes #12 .. #37 spun off for "more search tools"

screen shot 2017-09-13 at 2 56 44 pm

We may need to reassess how we want these things to collapse (BS3 vs BS4) but I think this gets us a MVP

### CODE REVIEW

File: app/views/quick_search/pages/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<% title %>
```
Comment (jcoyne) [inline app/views/quick_search/pages/home.html.erb]: Is this intentional?  You're calling a helper and not drawing anything?

File: app/views/quick_search/pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <%= image_tag 'handle.svg', class: 'img-fluid' %>
+  </div>
+  <div class='col-12 text-center'>
+      One search to see what's available in all these sources for your search topic.
+  </div>
+</div>
+<div class='row homepage-panels'>
+  <div class='col'>
+    <div class='card'>
+      <%= image_tag 'catalog.jpg', class: 'card-img-top' %>
```
Comment (jcoyne) [inline app/views/quick_search/pages/home.html.erb]: Do these images need alt text or `aria-hidden`?

File: app/views/quick_search/pages/home.html.erb (hunk 1 above)
Comment (mejackreed) [reply app/views/quick_search/pages/home.html.erb]: This a straight port from upstream.. so yep.

File: app/views/quick_search/pages/home.html.erb (hunk 2 above)
Comment (mejackreed) [reply app/views/quick_search/pages/home.html.erb]: This is implemented the same way as @jvine has specified in SearchWorks. By default they are getting an `alt=`

**Pull request merged**